includes_supermarket/includes_supermarket_private_fieri.rst

```diff
@@ -0,0 +1,34 @@
+.. The contents of this file may be included in multiple topics (using the includes directive).
+.. The contents of this file should be modified in a way that preserves its ability to appear in multiple topics.
+
+Fieri is an optional service what will check cookbook versions for certain metrics.  This helps to determine the quality of the cookbook.
+
+As of Supermarket 2.7 Fieri now lives within the Supermarket code base.
+
+If you are using a private |supermarket|, you can activate the Fieri service like this:
+
+First, add Fieri to your features attribute:
+
+.. code-block:: ruby
+
+   ['supermarket_omnibus']['config']['features'] = "tools,github,announcement,fieri"
+
+Then add the following Fieri attributes
+
+.. code-block:: ruby
+
+  ['supermarket_omnibus']['config']['fieri_key'] = "#{random string you generate}"
+  ['supermarket_omnibus']['config']['fieri_supermarket_endpoint'] = "#{your_supermarket_url}"
+  ['supermarket_omnibus']['config']['fieri_results_endpoint'] = "#{your_supermarket_url}/api/v1/cookbook-versions/evaluation
```

Comment:
Is `fieri_results_endpoint` included in the doc for compatibility with the 2.7.x versions, prior to 2.8.0 where we changed Fieri to use `fieri_supermarket_endpoint`?